src/app/about/page.tsx

```diff
@@ -1,23 +1,50 @@
 import { Container } from "@/components/shared/container";
 import { Metadata } from "next";
 
+interface SectionProps {
+  title: string;
+  children: React.ReactNode;
+
+}
+
 export const metadata: Metadata = {
   title: "About",
   description: "Learn more about the Legion of Mary",
 };
 
+const Section: React.FC<SectionProps> = ({ title, children }) => (
```

Comment:
what is going on here haha

Reply:
(as in what is this component for? Future use?)

Reply:
also TIL you can define props like this for a function

Reply:
Oh yes that will be a component for future use. And you never knew you could do that lol?